release-notes/6.0/preview/api-diff/preview1/Asp.Net/6.0-preview1_Microsoft.AspNetCore.Connections.Experimental.md

```diff
@@ -0,0 +1,35 @@
+# Microsoft.AspNetCore.Connections.Experimental
+
+``` diff
++namespace Microsoft.AspNetCore.Connections.Experimental {
```

Comment:
@halter73 @JamesNK @jkotalik is there a plan for this?

Reply:
Moving it out of experimental eventually, right?